Patches/ChatCommandPatch.cs

```diff
@@ -249,7 +249,8 @@ public static bool Prefix(ChatController __instance)
                         }
                         else Main.SetRoles[targetPc.PlayerId] = roleToSet;
 
-                        Utils.SendMessage("\n", localPlayerId, $"<b>{Utils.ColorString(Main.PlayerColors.TryGetValue(resultId, out var textColor) ? textColor : Color.white, targetPc.GetRealName())}</b>'s role in the next game will be <b><color={Main.roleColors[roleToSet]}>{GetString(roleToSet.ToString())}</color></b>");
+                       var playername = $"<b>{Utils.ColorString(Main.PlayerColors.TryGetValue(resultId, out var textColor) ? textColor : Color.white, targetPc.GetRealName())}</b>";
+                       var rolename = $"<color={Main.roleColors[roleToSet]}> {GetString(roleToSet.ToString())} </color>";                    

```

Comment:
Where did Utils.SendMessage() go?

Reply:
idk, this code was made by ph gaming, but when i tested setrole worked

Reply:
forgot to put here

Reply:
done